test/datatypes.test.js

```diff
@@ -206,7 +207,8 @@ describe('MySQL specific datatypes', function() {
     var xcor, ycor;
     City.create(city1, function(err, res) {
       if (err) return done(err);
-      res.loc.should.deepEqual(city1.loc);
+      const loc_in_geo_type = new GeoPoint(city1.loc);
+      res.loc.should.deepEqual(loc_in_geo_type);
```

Comment:
This change seems unrelated to the introduction of shared test suite from juggler v4. Is it needed to fix failing tests? Can we make it in a standalone PR please?